src/router_xor_filter_worker.erl

```diff
@@ -94,31 +147,62 @@ handle_cast(_Msg, State) ->
     lager:warning("rcvd unknown cast msg: ~p", [_Msg]),
     {noreply, State}.
 
+handle_info(?REBALANCE_FILTERS, #state{chain = Chain, oui = OUI} = State) ->
+    Pendings =
+        case router_console_api:get_all_devices() of
+            {error, _Reason} = Err ->
+                lager:error("failed to get device ~p", [_Reason]),
+                throw({could_not_rebalance_filters, Err});
```

Comment:
This will crash the server do we rly want this?

Reply:
no we do not. good eye